pages.de/common/eslint.md

```diff
@@ -0,0 +1,20 @@
+# eslint
+
+> Ein erweiterbarer Linter für JavaScript und JSX.
+> Weitere Informationen: <https://eslint.org>.
+
+- Erstellen einer ESLint Konfigurationsdatei:
+
+`eslint --init`
+
+- Linter auf mehreren Dateien ausführen:
+
+`eslint {{dateiname}}.js {{dateiname1}}.js`
```

Comment:
```suggestion
`eslint {{path/to/file1.js path/to/file2.js ...}}`
```

Mandatory fix (requires the [original](https://github.com/tldr-pages/tldr/blob/main/pages/common/eslint.md) page to be changed): [wrong](https://github.com/tldr-pages/tldr/blob/main/contributing-guides/style-guide.md#paths) path format is used, `eslint` doesn't require exactly two files to be passed.
Note: translation is based on google translate.

Reply:
wdym translation is based on google translate? you literally didn't translate it. also if you disagree with something in a translation, please open a separate pr fixing it in the original page

Reply:
Hm... Yes, my mistake. Thanks!

Reply:
I think `P`  and `D` mustn't be capital letters.

```suggestion
`eslint {{pfad/zu/datei1.js pfad/zu/datei2.js ...}}`
```